src/plugins/makeModifiers.js

```diff
@@ -23,7 +23,10 @@
  *   // => ['modifier']
  *
  */
-export const maker = (block, { modifier }, { delimiters }) =>
-  modifier && modifier.split(/\s+/).map((item) => `${block}${delimiters.modifier}${item}`)
+export const maker = (block, { modifier = '' }, { delimiters }) => {
+  const mod = modifier.trim()
+
+  return mod && mod.split(/\s+/).map(item => `${block}${delimiters.modifier}${item}`)
```

Comment:
When you have default value for `modifier` (for `trim` to exist) there is no need for the conditional (`mod &&`) and it can be chained:

```js
modifier
  .trim()
  .split(/\s+/)
  .map((item) => `${block}${delimiters.modifier}${item}`)
```